csi/exporter.py

```diff
@@ -67,12 +83,33 @@ def collect(self):
                 inodes_free_count.add_metric(labels, free)
                 inodes_used_count.add_metric(labels, used)
 
+                # Gathers capacity metrics for each subvol
+                pvcpath = pathlib.Path(os.path.join(pth, PV_TYPE_SUBVOL)).glob("*/*/*")
```

Comment:
- If PVCs are small in number it's fine to use `glob`, however i feel it may not scale well.
- If possible, pls amend [this](https://github.com/kadalu/kadalu/blob/b8e9c5fc70a3c5e52b21d006505e483b23c9b281/csi/volumeutils.py#L1225) function, but if it's too much to take leave it.

Reply:
I used directly the `yield_pvc_from_mntdir` function since we are already in a loop, adding the `path_prefix` to the returned list.

Reply:
Yep, good call.